python/ray/scripts/scripts.py

```diff
@@ -252,10 +252,14 @@ def start(node_ip_address, redis_address, address, redis_port,
         node_ip_address = services.address_to_ip(node_ip_address)
 
     if redis_address is not None or address is not None:
-        (redis_address, redis_address_ip,
-         redis_address_port) = services.validate_redis_address(
+        (redis_address, redis_address_ip, redis_address_port,
+         redis_password_found) = services.validate_redis_address(
```

Comment:
In what situation will I have a redis_password that I find here?

Like, I can only think of two cases:
1. redis is not started on the node yet, in which case there is no password.
2. redis is started on the node, in which case I die.